python3-build-requirements.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "python3-build-requirements",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"meson-python\" \"setuptools\" --no-build-isolation"
+    ],
```

Comment:
add a `"cleanup": ["*"]`

Reply:
This should be upstreamed to Flatpak Builder Tools instead...

Reply:
well it doesn't know what app you are working with, nor that these can be build deps of something. It might make sense to keep these for apps that are IDEs etc.

Reply:
Sorry, I thought you meant a different file.